Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -67,27 +63,28 @@ class UpstreamIntegrationTests: TestBase {
         resetTestExpectations()
         networkService.reset()
     }
-
+ 
```

Comment:
Are we running swift lint for integration tests as well? If not we can enable it for standard formatting across the repo.